circle.yml

```diff
@@ -13,14 +13,18 @@ machine:
 
 dependencies:
   post:
-    - pip install django
+    - pip install "django<2"
     - django-admin startproject testproject --extension py,yml,json --name Procfile,README.md --template=.
     - pip install requests pipenv --upgrade
 
 test:
   override:
     - pipenv install --dev:
         pwd: testproject
+    # The above command would have wrongly installed django 2.x
```

Comment:
I'm afraid it's not due to our Pipfile directly, but to dependencies installing it, in our case fixmydjango. @filipeximenes already merged my PR pinning a max version for this lib in there.

Reply:
It's a bug on pipenv, if we mention `django<2` it should not install another version of it, even if libs say wrongly that they work on `django 2`. Notice that if we moved `fixmydjango` to `[ packages]` instead of `[dev-packaged]` it would work just fine. IMO we should leave this workaround until this is fixed in pipenv (see issue number in comment) in case we add other libs that had similar constraints.